bundles/org.openhab.binding.mikrotik/README.md

```diff
@@ -0,0 +1,371 @@
+
+**All things provided by this binding requires a working bridge to be set up.**
+
+
+### Bridge Channels
+* <kbd>Number</kbd> `freeSpace` _(read-only)_ - Amount of free storage left on device in bytes
+* <kbd>Number</kbd> `totalSpace` _(read-only)_ - Amount of total storage available on device in bytes
+* <kbd>Number</kbd> `usedSpace` _(read-only)_ - Percentage of used device storage space
+* <kbd>Number</kbd> `freeMemory` _(read-only)_ - Amount of free memory left on device in bytes
+* <kbd>Number</kbd> `totalMemory` _(read-only)_ - Amount of total memory available on device in bytes
+* <kbd>Number</kbd> `usedMemory` _(read-only)_ - Percentage of used device memory
+* <kbd>Number</kbd> `cpuLoad` _(read-only)_ - CPU load percentage
+* <kbd>String</kbd> `uptime` _(read-only)_ - Duration while thing is up
+
+## WiFi Client Thing Configuration
+> Thing type: `wifiRegistration`
+
+Represents a wireless client connected to a RouterOS wireless network (direct or CAPsMAN-managed).
+
+The WiFi client thing configuration parameters are:
+
+* `mac` _(required text)_ - WiFi client MAC address
+* `ssid` _(optional text)_ - Constraining SSID for the WiFi client (optional). If client will connect to another SSID, 
+  this thing will stay offline until client reconnects to specified SSID.
+* `considerContinuous` _(optional integer)_ [Default: `180`] - The interval in seconds to treat the client as connected 
+  permanently. For example, when this set to 60, the binding will report a client away as soon 
+  as `lastSeen` + `60` seconds < `now`
+
+### WiFi client Thing Channels
+* <kbd>String</kbd> `macAddress` _(read-only)_ - MAC address of the client or interface
+* <kbd>String</kbd> `comment` _(read-only)_ - User-defined comment
+* <kbd>Switch</kbd> `connected` _(read-only)_ - Reflects connected or disconnected state
+* <kbd>Switch</kbd> `continuous` _(read-only)_ - Connection is considered long-running
+* <kbd>String</kbd> `ssid` _(read-only)_ - Wireless Network (SSID) the wireless client is connected to
+* <kbd>String</kbd> `interface` _(read-only)_ - Network interface name
+* <kbd>Number</kbd> `signal` _(read-only)_ - Received Signal Strength Indicator (RSSI) of the wireless client
+* <kbd>String</kbd> `uptime` _(read-only)_ - Duration while thing is up
+* <kbd>DateTime</kbd> `upSince` _(read-only)_ - Time when thing got up
+* <kbd>DateTime</kbd> `lastSeen` _(read-only)_ - Time of when the client was last seen connected
+* <kbd>Number</kbd> `txRate` _(read-only)_ - Rate of data transmission in megabits per second
+* <kbd>Number</kbd> `rxRate` _(read-only)_ - Rate of data receiving in megabits per second
+* <kbd>Number</kbd> `txPacketRate` _(read-only)_ - Rate of data transmission in packets per second
+* <kbd>Number</kbd> `rxPacketRate` _(read-only)_ - Rate of data receiving in packets per second
+* <kbd>Number</kbd> `txBytes` _(read-only)_ - Amount of bytes transmitted
```

Comment:
Number:DataAmount and not just Number if the channel uses UOM. BTW if you have created the items as Number, you need to delete and recreate them as Number:DataAmount to see the benefits of the built in graphing.

Reply:
@Skinah Does same applies to `Number:Dimensionless`? Also should percentage be declared as that UOM type?

Reply:
I can see a few possible ways those questions could be interrupted. Also please note I am not an expert in this and am still learning so best to ask some questions in the forum. In OH3 you can create an item that is Number:Dimensionless and also a Number so they are separate.

https://community.openhab.org/t/discussion-about-units-of-measurement/47183/5

Reply:
Yeah, let's keep percentage as `Dimensionless` then. Same mentioned in the forum thread you've specified above.